joint_trajectory_controller/test/joint_trajectory_controller_wrapping_test.cpp

```diff
@@ -0,0 +1,279 @@
+///////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2013, PAL Robotics S.L.
```

Comment:
Up to you. If it was on your own time, you can leave it like that or if it's within a company you may want to add them there. It's a test, hence it doesn't matter much.